llama_index/indices/prompt_helper.py

```diff
@@ -22,215 +25,138 @@ class PromptHelper:
     and fill in context information according to necessary token limitations.
 
     Args:
-        max_input_size (int): Maximum input size for the LLM.
-        num_output (int): Number of outputs for the LLM.
-        max_chunk_overlap (int): Maximum chunk overlap for the LLM.
-        embedding_limit (Optional[int]): Maximum number of embeddings to use.
-        chunk_size_limit (Optional[int]): Maximum chunk size to use.
+        context_window (int):                   Context window for the LLM.
+        num_output (int):                       Number of outputs for the LLM.
```

Comment:
i wonder if we should just put all three of these args in the service context (along with chunk size limit). 

Then 1) user basically never needs to specify prompthelper, which is good, 2) the chunk_overlap can also get passed to the text splitter, so users don't need to redefine a text splitter